x-pack/test/functional/services/ml/api.ts

```diff
@@ -634,5 +650,78 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
         }
       });
     },
+
+    async getAnnotations(jobId: string) {
+      log.debug(`Fetching annotations for job '${jobId}'...`);
+
+      const results = await es.search<Annotation>({
+        index: ML_ANNOTATIONS_INDEX_ALIAS_READ,
+        body: {
+          size: 1,
```

Comment:
I think line has to be removed or the value has to be increased as this method should fetch all annotations for a job, not just one.

Reply:
Updated in https://github.com/elastic/kibana/pull/73068/commits/f45a447e78d38637d6649fcb92e6d3235b13c906